tests/input/BINARY_OP.py

```diff
@@ -0,0 +1,25 @@
+(a, b) = (10, 3)
+print('Addition:', a + b)
+print('Subtraction:', a - b)
+print('Multiplication:', a * b)
+print('Division:', a / b)
+print('Modulo:', a % b)
+print('Floor Division:', a // b)
+print('Exponentiation:', a ** b)
+print('Greater than:', a > b)
+print('Less than:', a < b)
+print('Greater than or equal to:', a >= b)
+print('Less than or equal to:', a <= b)
+print('Equal to:', a == b)
+print('Not equal to:', a != b)
+print('Logical AND:', True and False)
+print('Logical OR:', True or False)
+print('Bitwise AND:', a & b)
+print('Bitwise OR:', a | b)
+print('Bitwise XOR:', a ^ b)
+print('Bitwise left shift:', a << b)
+print('Bitwise right shift:', a >> b)
+print('Membership in:', b in [1, 2, 3])
+print('Membership not in:', b not in [1, 2, 3])

```

Comment:
To avoid an optimization, you could change these to tuples, or just use `a in b` and `a not in b`...